### PR TITLE
[RFR] remove crimson hexagon references

### PR DESCRIPTION
**Pull request by jtotoole**
Decided to make a fresh PR rather than fight my way out of merge commit hell on https://github.com/mediacloud/backend/pull/785. 

Does adding `select insert_platform_source_pair( 'twitter', 'brandwatch' );` [here](https://github.com/mediacloud/backend/blob/master/apps/postgresql-server/schema/mediawords.sql#L1824) in `mediawords.sql` entail a migration, and should I also delete `select insert_platform_source_pair( 'twitter', 'crimson_hexagon' );`?

### CODE REVIEW

**Comment by jtotoole** [comment]
@pypt now we're in business (RFR = Ready For Review).

File: apps/crawler-fetcher/tests/python/test_podcast_resolve_url.py [hunk 1]
```diff
@@ -34,21 +34,21 @@ def test_get_feed_url_from_google_podcasts_url():
     assert _get_feed_url_from_google_podcasts_url('http://www.example.com/') == 'http://www.example.com/'
     assert _get_feed_url_from_google_podcasts_url('totally not an URL') == 'totally not an URL'
 
-    npr_feed_url = 'https://feeds.npr.org/381444908/podcast.xml'
```
Comment (jtotoole) [inline apps/crawler-fetcher/tests/python/test_podcast_resolve_url.py:37]: this feed url seems to oscillate at times for no reason, so hopefully the FT one will be more stable

**Comment by pypt** [comment]
> Does adding `select insert_platform_source_pair( 'twitter', 'brandwatch' );` [here](https://github.com/mediacloud/backend/blob/master/apps/postgresql-server/schema/mediawords.sql#L1824) in `mediawords.sql` entail a migration, and should I also delete `select insert_platform_source_pair( 'twitter', 'crimson_hexagon' );`?

I think it does - that function preloads the table with some platform-source pairs, so you might want to `DELETE` from the table that it preloads into to get rid of that deprecated platform-source pair.

**Comment by pypt** [comment]
> I think it does - that function preloads the table with some platform-source pairs, so you might want to DELETE from the table that it preloads into to get rid of that deprecated platform-source pair.

...and all of the rows in other tables with reference that particular row in `topic_sources`.

**Comment by jtotoole** [comment]
> ...so you might want to DELETE from the table that it preloads into to get rid of that deprecated platform-source pair

Would it be okay to do that in a separate PR once the pgmigrate one gets merged?

**Comment by pypt** [comment]
Very cool, thank you! Down with CH!

**Comment by pypt** [comment]
> Would it be okay to do that in a separate PR once the pgmigrate one gets merged?

Sure, moved this to https://github.com/mediacloud/backend/issues/798,

**Pull request merged**